src/Microsoft.AspNetCore.Server.Kestrel.Core/CoreStrings.resx

```diff
@@ -130,7 +130,7 @@
     <value>Chunked request incomplete.</value>
   </data>
   <data name="BadRequest_FinalTransferCodingNotChunked" xml:space="preserve">
-    <value>Final transfer coding is not "chunked": "{detail}"</value>
+    <value>The message body length cannot be determined because the final transfer encoding was set to '{detail}' instead of 'chunked'.</value>
```

Comment:
> If a Transfer-Encoding header field is present in a response and
       the chunked transfer coding is not the final encoding, the
       message body length is determined by reading the connection until
       it is closed by the server.  If a Transfer-Encoding header field
       is present in a request and the chunked transfer coding is not
       the final encoding, the message body length cannot be determined
       reliably; the server MUST respond with the 400 (Bad Request)
       status code and then close the connection.

Hmm, yup you're right. Ok to leave as encoding though? It makes more sense to me.